src/drive/mobile/actions/mediaBackup.js

```diff
@@ -98,20 +96,22 @@ export const startMediaBackup = (dir, force = false) => async (
   dispatch(endMediaUpload())
 }
 
-const uploadPhoto = (dirID, photo) => async (dispatch, getState) => {
+const uploadPhoto = (dirName, dirID, photo) => async (dispatch, getState) => {
+  let fileAlreadyExists = false
+  try {
+    await cozy.client.files.statByPath(dirName + '/' + photo.fileName)
+    dispatch(successMediaUpload(photo))
+    return
+  } catch (_) {} // if an exception is throw, the file doesn't exist yet and we can safely upload it
```

Comment:
Pre-emptive answer: this code is here and not in the part where we compute the list of files to upload because that would require checking all the files *before* starting the first upload, which can take a while.

Reply:
Puisque c'est mergé mais qu'on a discuté à l'oral, je note ici ma remarque : je pense que plus d'expressivité dans le code n'aurait fait de mal à personne.